mParticle-Pilgrim.xcodeproj/project.pbxproj

```diff
@@ -211,12 +212,32 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		5868204F21F27D4E00933B03 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "/usr/local/bin/carthage copy-frameworks\n";
```

Comment:
I believe this shouldn't be necessary for framework targets, please let me know if you know a reason why it's needed or otherwise please remove.

Reply:
I thought this was needed for carthage? That's what I was reading in some documentation

Reply:
Just confirmed in the main Carthage readme it's only needed for application targets:

```

Reply:
ahh gotcha, I'll remove it then!